### PR TITLE
Add magetab-curation-scripts

### PR DESCRIPTION
**Pull request by anjaf**
These scripts are an add-on package to the existing perl-atlas-modules. 

The scripts are wrappers to call main functionalities in the Perl modules and perform the following tasks:
- Validate MAGE-TAB format
- Apply curation and data file checks for ArrayExpress loading
- Apply curation checks for Expression Atlas loading
- Validate array design files (ADFs)
- Convert common array annotations into ADF format
- Manually insert MAGE-TAB submissions and ADFs into the submissions tracking system
- Run a tracking daemon to automatically trigger MAGE-TAB checks

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by pinin4fjords** [comment]
@BiocondaBot please add label

File: recipes/magetab-curation-scripts/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  sha256: 53e5cf855dea3d93dd8667c4e8f3effc02b856841c7d79041f4877ca2393e0e1
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  host:
+    - perl
+  run:
```
Comment (dpryan79) [inline recipes/magetab-curation-scripts/meta.yaml:18]: Can you move perl to a run-time requirement?

**Review by dpryan79 — CHANGES_REQUESTED** [review]
One small change

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by pinin4fjords** [comment]
Thanks for review @dpryan79

**Pull request merged**